static/map.js

```diff
@@ -1,43 +1,77 @@
-var map = null;
+"use strict";
 
-function initMap() {
-    // The location of Daytona
-    var location = {lat: 29.187668, lng: -81.072786};
-
-    // The map, centered at Daytona
-    map = new google.maps.Map(document.getElementById('map'), {zoom: 15, center: location});
-
-    var image = 'images/car.png'; // TODO: Add image for car
+// set initial values for time dropdowns
+function set_select(select, value) {
```

Comment:
This could also be accomplished server-side by putting a loop in the render template to create the dropdowns and having it add the "selected" attribute to the correct option